main.go

```diff
@@ -607,9 +701,16 @@ corectl eval by "Region" // Returns the values for dimension "Region"`,
 
 		PersistentPreRun: func(ccmd *cobra.Command, args []string) {
 			removeCmd.PersistentPreRun(removeCmd, args)
+			viper.BindPFlag("app", ccmd.PersistentFlags().Lookup("app"))
+			viper.BindPFlag("no-save", ccmd.PersistentFlags().Lookup("no-save"))
 		},
 
 		Run: func(ccmd *cobra.Command, args []string) {
+			if len(args) < 1 {
+				fmt.Println("Expected atleast one entity id specify what entity to remove from the app")
```

Comment:
This instruction is not really that clear :)

Reply:
fixed :)